crates/ide-completion/src/tests/special.rs

```diff
@@ -967,16 +967,10 @@ fn foo { crate:$0 }
 }
 
 #[test]
-fn no_completions_in_invalid_path() {
+fn no_completions_in_after_tripple_colon() {
     check(
         r#"
 fn foo { crate:::$0 }
-"#,
-        expect![""],
-    );
-    check(
-        r#"
-fn foo { crate::::$0 }
```

Comment:
Ah wait, we no longer catch this because the token here is a `::` again right? Let's add that to the check as well as it is simple, I forgot that :) That is, `t.kind() == T![:] || t.kind() == T![::]` for the previous token check

Reply:
done 1ca5cb7ed9928fad4222cb26727da41ff9ac148f